app/models/search_query.rb

```diff
@@ -8,8 +8,14 @@ class SearchQuery < ApplicationRecord
 
   def self.import(file, user)
     keywords = []
-    CSV.foreach(file.path, headers: true) do |row|
-      keywords << row[0]
+    CSV.foreach(file.path) do |row|
+      if row.count > 1
+        row.each do |keyword|
+          keywords << keyword
+        end
+      else
+        keywords << row[0]
+      end
     end
```

Comment:
There needs to also be a validation for not allowing more than 100 keywords as per the requirements